.gitignore

```diff
@@ -4,6 +4,9 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
+# generated files
+db.schema.rb
```

Comment:
This shouldn't be in .gitignore because you do want to commit it sometimes, just only when you're actually making database changes.

i.e. if you've written a migration that changes the schema, you should also commit the schema. But if your local schema has just changed because your local schema has drifted out of sync with what migrations create you shouldn't.

Reply:
reverted